packages/i18n/src/messages/errorMessages.ts

```diff
@@ -116,6 +116,18 @@ const errorMessages = defineMessages({
     id: "Gv3lBS",
     description: "Message for Unauthorized or Request Rejected server response",
   },
+  downloadRequestFailed: {
+    defaultMessage:
+      "There was a problem downloading your file.  If you continue to receive this error, please get in touch with our support team.",
```

Comment:
Extra space between `file. If`.

Reply:
Fixed in [fix typos](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10992/commits/03a84f9ea9c3dadcc062a72d6e6a27407af29a99)